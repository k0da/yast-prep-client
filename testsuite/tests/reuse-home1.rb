# encoding: utf-8

# testedfiles: helper1b.yh
module Yast
  class ReuseHome1Client < Client
    def main
      Yast.include self, "setup-system.rb"

      setup_system("reuse-home")

      Yast.include self, "helper1a.rb"

      Yast.import "ProductFeatures"

      ProductFeatures.SetBooleanFeature("partitioning", "try_separate_home", true)
      ProductFeatures.SetBooleanFeature("partitioning", "proposal_lvm", false)
      ProductFeatures.SetBooleanFeature("partitioning", "proposal_snapshots", false)
      ProductFeatures.SetStringFeature("partitioning", "root_max_size", "20 GB")
      ProductFeatures.SetStringFeature("partitioning", "root_base_size", "15 GB")

      Yast.include self, "helper1b.rb"

      nil
    end
  end
end

Yast::ReuseHome1Client.new.main
