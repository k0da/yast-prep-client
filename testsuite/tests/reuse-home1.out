Dump	Proposal:
Dump	Format partition /dev/sda2 (15.01 GiB) with btrfs [destructive]
Dump	Create subvolume boot/grub2/i386-pc on device /dev/sda2
Dump	Create subvolume opt on device /dev/sda2
Dump	Create subvolume srv on device /dev/sda2
Dump	Create subvolume tmp on device /dev/sda2
Dump	Create subvolume usr/local on device /dev/sda2
Dump	Create subvolume var/crash on device /dev/sda2
Dump	Create subvolume var/lib/mailman on device /dev/sda2
Dump	Create subvolume var/lib/named on device /dev/sda2
Dump	Create subvolume var/lib/pgqsl on device /dev/sda2
Dump	Create subvolume var/log on device /dev/sda2
Dump	Create subvolume var/opt on device /dev/sda2
Dump	Create subvolume var/spool on device /dev/sda2
Dump	Create subvolume var/tmp on device /dev/sda2
Dump	Use /dev/sda1 as swap
Dump	Set mount point of /dev/sda3 to /home
Dump	
Dump	Target Map Excerpt:
Dump	device:/dev/sda label:msdos
Dump	device:/dev/sda1 fsid:0x82
Dump	device:/dev/sda2 fsid:0x83
Dump	device:/dev/sda3 fsid:0x83
